### ISSUE TITLE
Tumblr blogs immune

### ISSUE THREAD

**Issue by maxoloti** [shinigami-eyes/shinigami-eyes#86]
A select few Tumblr blogs appear to be immune. I have found 2 of these blogs, and know that I am marking them through means that will work for other blogs. I believe this error is because the URLs are formatted as "https://tumblr.com/username" as opposed to the standard "https://username.tumblr.com". I have tried repeatedly and have found no possible way to mark these blogs, as they will always be recognised by the extension as "tumblr.com" rather than an individual.

**Comment by citrusella** [comment]
They're login-required blogs, but sometimes if you open their full blog up their username at the top of a post they make will still be in name.tumblr.com structure and thus markable.

**Comment by maxoloti** [comment]
Yeah I tried that and it wasn't the case for these blogs. It redirects immediately to tumblr.com/name (I assume they don't have a theme?), and all possible links continue in that same format. The setting to require a log-in was my first theory on why this was happening, but I tried in another browser where I'm not logged in and had no trouble accessing the blog, but still couldn't mark it. I even tried typing the link in the correct format and marking that and it seems like it would be marked from say, looking them up on Google. I do believe they are hidden from search results anyway, however.

**Comment by ShieldOfWesnoth** [comment]
I was just about to report this error, too!  Attempting to flag a "dashboard view only" tumblr blog results in Shinigami Eyes telling you thusly:

**This item could not be labeled. Possible reasons:**
**- It doesn't represent a specific user or page.**
**- It's not a kind of object supported by Shinigami Eyes.**

**www.tumblr.com**

A similar issue sometimes occurs now when viewing _any_ tumblr blog in dashboard view, but blogs viewable on desktop as well can still also be reported to Shinigami Eyes in dashboard view with some careful clicking (specifically, hovering over the right links)...  This is a _relatively_ new issue, thanks to Tumblr's _relatively_ recent decision to allow many blogs-- even "main" ones-- to be only viewable in dashboard, so it's not a huge surprise it hasn't been nipped yet.  However, if making your blog dashboard-view-only is all it takes to dodge Shinigami Eyes, it's a huge hole through which TERFs, truscum, and other transphobes can leap and avoid being flagged.  It may seemingly only effect logged-in tumblr users who could view such blogs in the first place, but _most_ people who view tumblr posts are logged in, posts from transphobes can still make it onto desktop-viewable blogs, and it's all too easy for a violently transphobic person's comment on an innocent viral post to get spread widely.  This is a huge problem, and I hope it'll be a quick and easy fix.

**Comment by CharaViolet** [comment]
fr I was about to report this, like, tumblr user transmysoginist is a massive terf (not just because of that URL) yet they're completely unmarked and I can't mark them because of this. i'd say this is a very high priority update imo

**Comment by fishsticxz** [comment]
same issue here, trying to report the-rad-menace and you-dont-pass among others, because there's a funny post circulating by them and nobody seems to read urls anymore. i feel like the terfs and other transphobes are taking advantage of not being able to be marked on shinigami eyes by only using the dashboard view blog, hope this gets fixed soon. i'm pretty sure you were able to report people from their replies before too, but that doesn't work anymore either

**Comment by SpacetimeMeatMachine** [comment]
I have also noticed this, and absolutely believe that they are taking advantage of the dashboard-view-only layout as well as taking their blog URLs off their headers, perhaps with some kind of formatting quirk. 
I have included screenshots here, including a scrap of formatting in the header without the URL that may be a clue:
![1](https://user-images.githubusercontent.com/116672115/197845777-b8342705-5eb4-4c50-8641-967b8b185413.PNG)
![3](https://user-images.githubusercontent.com/116672115/197845801-494c1c01-282d-4673-8416-69a15a3f4709.PNG)

Regardless, this bug is rapidly making the extension largely useless on Tumblr, and the TERFs seem to be emboldened by this as evidenced by the recent explosion of outright hateful accounts who are putting 'radfem' or 'terf' right in their URLs.

**Comment by 6ftslug** [comment]
I'm just commenting to hopefully add traction to this issue. Tumblr user swag-fem is also using this tactic, and them and other tumblr radfems are even getting bold enough to start sending out open hate and s--cide bait again (ask me how i know lol) as well as reblogging harassment. Sad to see no one has found a workaround :( 
We'd all really appreciate a patch to this otherwise perfect extension! <3

**Comment by citrusella** [comment]
It's possible with the 15 months since the last update that the extension is becoming abandonware (and there is no clearly-actively-maintained, trustworthy fork), so I don't know if the person who made this would come back and patch this.

But at the very least talking about it makes it known.

**Comment by lunarna-gh** [comment]
Now updated to 1031 on Firefox store, and it's marked as "released under the MIT license". Extension has received no commits on GitHub. The email listed on the Chrome store doesn't reply when asked for clarification about the license.

**Comment by wolfcat-hybrid** [comment]
Sorry for commenting on an older thread, but it seems to have been fixed? Still no update from the shinigami eyes dev as far as I can see, so it seems Tumblr might have done something on their end. I don't know much about coding, was wondering if anyone here knew what might have happened.

**Comment by citrusella** [comment]
The extension was updated without the code in Github being updated, and one intent of that update was specifically to fix Tumblr.
